APP-2922-sym-parser-emojis-bug Fixed emojis bug

el-youness opened the pull request:
Done by checking the existence of the 'id' parameter in the 'data' node of the payload.

symphony-youness committed:
APP-2922-sym-parser-emojis-bug Fixed emojis bug

Done by checking the existence of the 'id' parameter in the 'data' node of the payload.

symphony-youness committed:
APP-2922-sym-parser-emojis-bug removed unused code block

symphony-hong commented on sym_api_client_python/processors/sym_message_parser.py:
```diff
@@ -60,6 +60,14 @@ def get_im_name(self, message_data):
     def get_stream_id(self, message_data):
         return message_data['stream']['streamId']
 
+    def get_tags(self, json_nodes, tag_type):
```
I would make this method private `def _get_tags(self, json_nodes, tag_type):` like what has been done in Java sdk: [getTags](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/6fd04cd6f09ffa10d8926667cfe350e570bd6bc0/symphony-bdk-legacy/symphony-api-client-java/src/main/java/utils/SymMessageParser.java#L72)

el-youness replied:
done.

symphony-hong commented on sym_api_client_python/processors/sym_message_parser.py:
```diff
@@ -60,6 +60,14 @@ def get_im_name(self, message_data):
+        tags = []
+        if json_nodes:
+            for k, v in json_nodes.items():
+                if 'id' in v and len(v['id']) > 0 and v['id'][0]['type'] == self.MENTION_TYPE:
```
the condition should be: `v['id'][0]['type'] == tag_type`, isn't it?

symphony-hong replied:
do you think we need to check: `'type' in v['id'][0]`? (and `value` as well)

el-youness replied:
Fixed.
For the question. I do not know the purpose of those conditions but I added them anyways to stay consistent with the Java SDK.

symphony-hong reviewed (changes requested):
Some comments added

symphony-youness committed:
APP-2922-sym-parser-emojis-bug quick fix
- set method to private
- added same conditions to filter json nodes to match the java sdk guidelines

symphony-hong reviewed (approved):
LGTM!

symphony-elias reviewed (approved):
LGTM, maps the logic in legacy java SDK
